libs/hdf-converters/src/asff-mapper/case-hdf2asff.ts

```diff
@@ -21,13 +21,18 @@ function objectifyTypesArray(
   if (!Array.isArray(typesArray)) {
     typesArray = _.get(typesArray, 'FindingProviderFields.Types') as string[];
   }
+  const groupedTypes = _.groupBy(typesArray, (typeString) =>
```

Comment:
Consider changing the name of the file to case-what-the-thing-is, maybe `case-converted-from-hdf`, or `case-previously-hdf`.

Reply:
went with previously-hdf